### ISSUE TITLE
Install and update via winget

### ISSUE THREAD

**Issue by gohma231** [stevencohn/OneMore#1019]
# New Feature/Solution

Installing and updating OneMore can be made easier using the Microsoft winget tool.

https://github.com/microsoft/winget-cli

THE FINE PRINT This is a hobby project that I started for myself. I'm happy to take requests and will promise to
consider each one carefully but offer no guarantees that I will ultimately agree to their
usefulness or have time to implement any of them. If OneNote offers a "close enough" work-around
then I will probably reject the request - you've been warned!

**Comment by maphew** [comment]
v5.9 is available via Winget but newer releases are not. I looked around a bit and it seems that new releases are supposed to be updated automatically but I haven't found the trigger point. Maybe it needs to be maintainer triggered?

* https://github.com/microsoft/winget-pkgs/pull/116632
* https://github.com/vedantmgoyal2009/vedantmgoyal2009/blob/main/winget-pkgs-automation/packages/s/stevencohn.onemore.json

**Comment by stevenmcohn** [comment]
The community has been publishing OneMore to winget :heart:

**Comment by stevencohn** [comment]
The community has been publishing OneMore to winget :heart:

**Comment by maphew** [comment]
this looks to be a straightfroward way to add here so that winget releases happen at the source: 
https://github.com/marketplace/actions/winget-releaser
I started down the road of making a fork with intention to issue a PR with this, then saw a personal access token is required. I think that means it would break or not work outside my fork (?)

**Comment by stevencohn** [comment]
Correct. You'd be responsible for creating and managing your own releases. As you publish each release, it uses your PAT to publish to winget-pkgs. To be clear, it's not a PR (pull request) but rather a Release Publish.

**Comment by maphew** [comment]
I went through the process anyway, to see what it looked like. There was a checkbox to "share secret with repository owner" so maybe it might work after all? (if you find the change worthy and merged the pull request that is).

**Comment by Dragon1573** [comment]
Excuse me, @maphew @stevencohn. Sorry for digging a stale issue. 🙏🏼

I notice the [Publish to WinGet](https://github.com/stevencohn/OneMore/actions/workflows/winget-releaser.yml) workflow is disabled. Could I ask for an explanation? What can I do for re-enable the workflow?

- Related microsoft/winget-pkgs#265751
- Related microsoft/winget-pkgs#265995

> [!NOTE]
> 
> There's no [stevencohn/winget-pkgs](https://github.com/stevencohn/winget-pkgs) repository fork under @stevencohn's account. Also there might be a missing `WINGET_TOKEN` Repository Secret (a **classic** GitHub Personal Access Token) with `public_repo` scope for the workflow to run properly.

**Comment by stevencohn** [comment]
Honestly, I tried but couldn't get it working! I think it may be easier if the MSI was signed. I'm working on that under #1832

**Comment by UnownPlain** [comment]
The workflow probably doesn't work due to a bug in the rust-msi crate that komac (the underlying tool that Winget Releaser runs to generate and submit manifests) uses.

mdsteele/rust-msi#19

![Image](https://github.com/user-attachments/assets/64bf74e6-b3e5-4463-86c0-8f2b1c8a7a45)

**Comment by UnownPlain** [comment]
This issue should be resolved now with the newest komac release.

> The workflow probably doesn't work due to a bug in the rust-msi crate that komac (the underlying tool that Winget Releaser runs to generate and submit manifests) uses.
> 
> [mdsteele/rust-msi#19](https://github.com/mdsteele/rust-msi/issues/19)
> 
> ![Image](https://github.com/user-attachments/assets/64bf74e6-b3e5-4463-86c0-8f2b1c8a7a45)
